### ISSUE TITLE
TabMon Does Not Handle Backgrounder Nodes Well

### ISSUE THREAD

**Issue by danjrahm** [tableau/TabMon#201]
Quote directly from hh9999 in issue #199.

"""
I experienced the same issue (running a backgrounder host). It appears that all of the processes for each host should be defined in the config file. For example:

```
 <Host computerName="COMPUTER_NAME_HERE" address="ADDRESS_HERE" specifyPorts="true">
    <ProcessType processName="dataserver">
    </ProcessType>
    <ProcessType processName="vizql">
    </ProcessType>
    <ProcessType processName="background job monitor">
         <Process portNumber="PORT#" processNumber="0"/>
         <Process portNumber="PORT#" processNumber="1"/>
         <Process portNumber="PORT#" processNumber="2"/>
         <Process portNumber="PORT#" processNumber="3"/>
         <Process portNumber="PORT#" processNumber="4"/>
    </ProcessType>
    <ProcessType processName="vizportal">
    </ProcessType>
  </Host>
```

This allowed the service to start correctly. Also to note if you use 2018.x, some of the process names are prefixed with "run-" so you'll need to account for those in the sample workbook (Tableau Service dimension) or you won't see the data.
"""

**Comment by danjrahm** [comment]
I don't think this is a TabMon Config Builder issue. Instead it is an issue with how we build out our counters. This should be resolved with my next release.

**Comment by danjrahm** [comment]
This is fixed in [v1.4](https://github.com/tableau/TabMon/releases/tag/v1.4) release.
